tests/system/python/conftest.py

```diff
@@ -215,6 +215,25 @@ def clone_make_install():
 
     return _add_fledge_north
 
+
+@pytest.fixture
+def update_stat_collection(fledge_url, wait_time):
+    """Update the Stat colectioin of all south service to per asset & service"""
+    
+    # Wait for the south service to be created
+    time.sleep(wait_time)
```

Comment:
Remove extra delay from here. This fixture should call when service is up

Reply:
We can call once added in disable mode as well; and advance category is available? So not necessarily when up.